docs/marionette.application.md

```diff
@@ -160,6 +160,21 @@ allows you to provide extra configuration for various parts of your app througho
 initialization sequence.
 
 ```javascript
+var App = Mn.Application.extend({
+  initialize: function(options) {
+    console.log('Initialize');
+  },
+
+  onBeforeStart: function(app, options) {
+    this.model = new MyModel(options.data); 
```

Comment:
One more. If we are removing arguments `onBeforeStart: function(options) {` in example above, it will be nice to be consistent and use `this.options.data` here to.